docs/avro.rst

```diff
@@ -145,22 +98,94 @@ simply dropped.  Had the new schema dropped the original field ``favorite_number
 forward compatible with the original user schema since we wouldn't know how to fill in the value for ``favorite_number``
 for the new data because the original schema did not specify a default value for that field.
 
+If the compatibility is set to ``FORWARD_TRANSITIVE`` (not just ``FORWARD``), then it means that consumers using all previousely registered schemas (not just the latest schema) can read data written by producers using the new schema.
+
 
 .. _avro-full_compatibility:
 
 Full Compatibility
 ^^^^^^^^^^^^^^^^^^
 
-Full compatibility means schemas are backward **and** forward compatible.
+``FULL`` compatibility means schemas are backward **and** forward compatible.
```

Comment:
How about this: FULL compatible schema means that both of these are true:
1. Version X-1 of the schema can be used to decode data encoded with version X of the schema, but not necessarily version X-2 and before.
2. Version X of the schema can be used to decode data encoded with version X and X-1 of the schema, but not necessarily version X-2 and before.

In other word, FULL compatibility means schemas are backward and forward compatible.

Reply:
Good idea. I've adapted this suggestion